### ISSUE TITLE
When spamming buttons while mapping .exe freezes or crashes

### ISSUE THREAD

**Issue by DanSucksAtExploitingInRoblox** [David20122/Sound-Space-Quantum-Editor#10]
I was making a impossible map but at the start it just crashed

**Comment by krmeet** [comment]
If by "crashed" you mean it stopped responding, you should be able to just wait for it to respond again as the issue would be you spamming too much and trying to make the editor perform too many tasks at once. Could you provide screenshots or any other useful information before this issue gets closed?

**Comment by DanSucksAtExploitingInRoblox** [comment]
Yes, it did stop responding. but after it continued responding it just closes itself/
